controllers/backplaneconfig_controller_test.go

```diff
@@ -252,6 +252,12 @@ var _ = Describe("BackplaneConfig controller", func() {
 				ResourceType:   clusterManagementAddon,
 				Expected:       nil,
 			},
+			{
+				Name:           "Cluster Proxy Addon Manager",
+				NamespacedName: types.NamespacedName{Name: "cluster-proxy-addon-manager"},
```

Comment:
One of the deployments is missing, and since these are deployments they need a namespace